includes/admin/payments/orders.php

```diff
@@ -529,7 +529,7 @@ function edd_order_details_items( $order ) {
 			<div class="edd-add-download-to-purchase">
 				<ul>
 					<li class="download">
-						<span class="edd-order-details-label-mobile"><?php printf( esc_html_x( '%s To Add', 'order details select item to add - mobile', 'easy-digital-downloads' ), edd_get_label_singular() ); ?></span>
+						<label for="edd_order_add_download_select_chosen" class="edd-order-details-label-mobile"><?php printf( esc_html_x( '%s To Add', 'order details select item to add - mobile', 'easy-digital-downloads' ), edd_get_label_singular() ); ?></label>
```

Comment:
The select ID for this is actually `edd_order_add_download_select`, looking at the markup. It's weird because the ID being passed to the `product_dropdown` method is `edd-order-add-download-select`, but it looks like the method is overriding/replacing that. And then we have the `chosen` issue as well, but I think our label should match the `select` element we are actually creating.